drupal/recipes/wunder_pages/config/field.field.node.page.field_image.yml

```diff
@@ -0,0 +1,45 @@
+uuid: 22892441-eb73-40d2-8b4a-89cb66e65c27
+langcode: fi
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_image
+    - node.type.page
+  module:
+    - content_translation
+    - image
+third_party_settings:
+  content_translation:
+    translation_sync:
+      alt: alt
+      title: title
+      file: '0'
+id: node.page.field_image
+field_name: field_image
+entity_type: node
+bundle: page
+label: 'Search/Meta Image'
+description: 'Image used in the search results and for Open Graph html meta tags, so that sharing the content looks good.'
```

Comment:
The search results do not display images at the moment :)

Reply:
Is it coming soon, or am I just predicting into the wild :D but isn't it though? In eg. google searches :)

Reply:
it could be a good next feature for the search actually :)